pom.xml

```diff
@@ -114,6 +118,11 @@
                             <artifactId>maven-failsafe-plugin</artifactId>
                             <version>3.0.0-M6</version>
                         </plugin>
+                        <plugin>
+			                <groupId>org.apache.maven.plugins</groupId>
+			                <artifactId>maven-enforcer-plugin</artifactId>
+                            <version>3.0.0</version>
```

Comment:
> When I first started working on the project there were conflicts with the state of my local repository as the cucumber project did not have any minimum requirements. The unmanaged enforcer plugin was not API compatible with my newer maven version.

The enforcer plugin _is managed_ through the parent pom and needs an upgrade there, not in this project: https://github.com/cucumber/cucumber-parent/pull/25. Please remove this.

Reply:
Unfortunately I didnt see the plugin management of the enforcer in the parent. Did not notice that there is a parent outside the project.